app/src/main/res/layout/dialog_pdf_export.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
Why do you need to duplicate similar layout for CSV layout? You need to change only title